src/pages/ReportDetailsPage.tsx

```diff
@@ -428,12 +429,23 @@ function ReportDetailsPage({policies, report, session, personalDetails}: ReportD
                 action: () => unapproveExpenseReportOrShowModal(),
             });
         }
+
+        if (report?.reportID && isDebugModeEnabled) {
+            items.push({
+                key: CONST.REPORT_DETAILS_MENU_ITEM.DEBUG,
+                translationKey: 'debug.debug',
+                icon: Expensicons.Bug,
+                action: () => Navigation.navigate(ROUTES.DEBUG_REPORT.getRoute(report.reportID)),
+                isAnonymousAction: true,
+                shouldShowRightIcon: true,
+            });
+        }
+
         return items;
     }, [
         isSelfDM,
         isArchivedRoom,
         isGroupChat,
-        isRootGroupChat,
```

Comment:
why all these changes to the order?

Reply:
No clue, but I'm reverting these changes!